### ISSUE TITLE
How to train malheur

### ISSUE THREAD

**Issue by alxbob** [spender-sandbox/cuckoo-modified#355]
I couldn't find any instructions on how to train malheur beyond cuckoo-modified train set. Any instructions how to's? Or malheur is autotrained?

**Comment by doomedraven** [comment]
Automated

**Comment by alxbob** [comment]
So how malheur nows and something malicious and something that is not malicious? So i auto trade malheur by feeding cuckoo with samples?

**Comment by KillerInstinct** [comment]
Automated in the sense that when you submit a sample and it doesn't match similar behaviors of any previous samples, it won't have any similar analysis. Early on when you only have a couple hundred analyses, you have may some FP's -- but the best way to train it is to give it samples you want it to learn. If the first sample you submitted was locky, and the next day you got a new campaign, there's a good chance that the second analysis would have the first one as a 'similar' analysis, even if you only had one previous sample.

**Comment by alxbob** [comment]
Thanks for your answer
What i don't understand is that if i commit a sample with a zeroday malware or a custom metepreter that doesn't have a know signature (and 0 score) there is no way to tell malheur that it is a malware and train it to find next samples with similar behavior right? I have to do it custom some way or someone must classified it as malicious so malheur would learn...

**Comment by doomedraven** [comment]
@alxbob i would you suggest to read malheur plugin to understand how it works

**Comment by alxbob** [comment]
Yes i have to...i asked before start reading further for some generic answers from ppl that are more informed than me! So an onliner would be helpful also some links for further studying. Thanks for all your answers

**Comment by alxbob** [comment]
So the short answer is that training is automated and long answer is that zerodays should manually tagged somehow and then malheur should learn about them?

**Comment by Mokatsu** [comment]
This might not be the place to ask this but, is anyone using Malheur in a non-modified cuckoo version?

**Comment by doomedraven** [comment]
lol isnt' the place, and no
